systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaMirrorMakerResource.java

```diff
@@ -24,6 +25,7 @@
 
 public class KafkaMirrorMakerResource {
     public static final String PATH_TO_KAFKA_MIRROR_MAKER_CONFIG = "../examples/mirror-maker/kafka-mirror-maker.yaml";
+    private static final long READINESS_TIMEOUT = ResourceOperation.getTimeoutForResourceReadiness();
```

Comment:
Are you using this instance variable only inside `deployKafkaMirrorMaker` method?

Reply:
Yes, but it's done like this if we want to add something else, otherwise I can add it straight to the `waitFor` -> and also to match the pattern to have it like this in all resource classes